tools/esdlcmd/README.md

```diff
@@ -0,0 +1,485 @@
+                <LogDataXPath>
+                    <LogInfo name="PreparedData" xsl="log-prep">
+                </LogDataXPath>
+                <XSL>
+                    <em:Transform name="log-prep">
+                        <em:Include file="log-prep.xslt">
+                    </em:Transform>
+                </XSL>
+            </LogAgent>
+        </LoggingManager>
+    </em:ServiceBinding>
+    <em:EsdlDefinition>
+        <em:Include file="WsFoobar.ecm"/>
+    </em:EsdlDefinition>
+</em:Manifest>
+```
+Because the tool is permissive and flexible, copying through all markup not in the manifest namespace, the manifest elements are only required in order to take advantage of the automated processing and simplified format of the manifest file. This example highlights the recommended usage of manifest elements to use the tool's capabilities. Although you could replace some of the elements below with verbatim `bundle` or `binding` output elements we won't cover that usage here.
+
+* `<em:Manifest>` Is the required root element. By default the tool outputs a `bundle`, though you may explicitly override that on the command line or by providing an `@outputType=binding` attribute.
+* `em:ServiceBinding>` is valid for both `bundle` and `binding` output. It is necessary to enable recognition of `<em:Scripts>`, and `<em:Transform>` elements.
+* `<em:EsdlDefinition>` is relevant only for `bundle` output. It is necessary to enable element order preservation and recognition of `<em:Include>` as a descendant element.
+* `<em:Include>` causes external file contents to be inserted in place of the element. The processing of included files is context dependent- the parent of the `<em:Include>` element dictates how the file is handled. File inclusion facilitates code reuse in a configuration as code development environment.
+* `<em:Scripts>` and `<em:Transform>` trigger preservation of element order for all descendent elements and enable `<em:Include>` recognition. XML element ordering is significant to proper ESDL script, XSLT, and ESXDL content processing. The platform's `IPropertyTree` implementation, used when loading an artifact, does not preserve order. Output configuration files must embed this content as text as opposed to XML markup. The tool automates the conversion of XML markup, which is easier for a configuration author to create and maintain, into the embedded text required by an ESP.
+
+### Syntax
+
+These elements may create artifact content, change the tool's behavior, or both. When used as intended, none of these elements will appear in the generated output:
+
+#### Manifest
+Required root element of all manifest files that create output:
+
+- `bundle` output is created by default. It can be made explicit by setting either command line option `--output-type` to `bundle` and/or `@outputType` is `bundle`.
+- `binding` output is created when either command line option `--output-type` is `binding` and/or `@outputType` is `binding`.
```

Comment:
Suggest a minor update to phrasing - "either command line option ..." can be (mis)interpreted to consider "@outputType" to be a command line option. Maybe "either the command line option ... and/or the manifest property ...".

Reply:
clarified